samples/smoketest/Program.py

```diff
@@ -0,0 +1,13 @@
+from KeyVaultSecrets import KeyVault
```

Comment:
This program depends on some set of packages. To my knowledge those are tracked in some kind of `requirements.txt` file.